docs/index.html

```diff
@@ -14,6 +14,16 @@
         font-weight: normal;
       }
       .remark-code, .remark-inline-code { font-family: 'Ubuntu Mono'; }
+
+      .left-column {
+        float: left;
+        width: 49%;
+      }
+
+      .right-column {
+        float: right;
+        width: 49%;
```

Comment:
`.left-column` and `.right-column` css rules allow for writing slides that have two sections side by side from each other, like slide 28 of this presentation shows.

---

![two-columns](https://user-images.githubusercontent.com/2654835/71649885-335bf500-2cc7-11ea-8fde-987a66f56fb8.png)

Reply:
👍